docker/openemr/7.0.5/kcov-wrapper.sh

```diff
@@ -1,15 +1,53 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
I don't really mind if we want to use bash, but I don't see anything bash-specific in the script.